src/editor/Core/Theme.re

```diff
@@ -22,6 +22,20 @@ module EditorColors = {
     editorMenuForeground: Color.t,
     editorMenuItemSelected: Color.t,
     tabActiveForeground: Color.t,
+    statusBarItemVisualModeBackground: Color.t,
```

Comment:
Thanks for adding the mode colors and defaults! There's no real equivalent in VSCode so I think we will definitely need our own.

I'm wondering if we could generalize this to something like `oniVisualModeBackground`  (using the 'oni' prefix as a convention for theme colors that aren't available in VSCode). 

Seeing this change made me want these colors on top of the active tab (like we did in Oni1), so I was thinking there'd be a case where we might re-use these colors in other parts of the UI, too.